ntp-daemon/src/main.rs

```diff
@@ -28,70 +40,96 @@ async fn main() -> Result<(), Box<dyn Error>> {
 
     let mut snapshots = Vec::with_capacity(peers.len());
 
+    // when we perform a clock jump, all current measurement data is off. We force all associations
+    // to clear their measurement data and get new data. This vector contains associations that
+    // have not yet confirmed that their measurement data has been cleared.
+    let mut waiting_for_reset: Vec<PeerChannels> = Vec::with_capacity(peers.len());
```

Comment:
so basically  the reset process is to move all peers into this vector `waiting_for_reset.append(&mut peers)` and triggering the reset of the peers `_reset_tx.send(...)`. The rest (resetting the peer, reinstating the peer as giving valid measurements) is already handled in this PR.